ambry-api/src/main/java/com/github/ambry/server/storagestats/ContainerStorageStats.java

```diff
@@ -148,6 +156,17 @@ public Builder(short containerId) {
       this.containerId = containerId;
     }
 
+    /**
+     * Constructor to instantiate a new {@link Builder} with the given {@link ContainerStorageStats}.
+     * @param origin The given {@link ContainerStorageStats}.
+     */
+    public Builder(ContainerStorageStats origin) {
```

Comment:
nit: Do we need both the Builder and the copy constructor? Looks like they are doing the same thing.